src/helpers.js

```diff
@@ -65,6 +66,34 @@ export default {
       allureReporter.addAttachment('Execution video', videoPath, 'video/mp4');
     }
 
+    const frames = glob.sync(`${this.recordingPath}/*.png`);
+
+    if (frames.length) {
+      const frameRegex = /^.*\/(\d{4})\.png$/;
```

Comment:
Let's move this to the top of the file, it doesn't need to be re-declared every time